src/fiskaltrust.ifPOS/v1/it/IITSSCD.cs

```diff
@@ -44,5 +44,11 @@ public interface IITSSCD
         /// </summary>
         [OperationContract(Name = "v1/NonFiscalReceipt")]
         Task<Response> NonFiscalReceiptAsync(NonFiscalRequest request);
+
+        /// <summary>
+        /// This endpoint offers a generic option to process the given receiptrequest / receiptresponse pair. The specific SCU implementation will decide which receipt type to process.
+        /// </summary>
+        [OperationContract(Name = "v1/Process")]
+        Task<ProcessResponse> ProcessAsync(ProcessRequest request);
```

Comment:
@TSchmiedlechner do you know how launcher handles if this method is not existent in a Queue?